src/hypertrace/agent/instrumentation/django/__init__.py

```diff
@@ -0,0 +1,56 @@
+'''Hypertrace django instrumentor module wrapper.''' # pylint: disable=R0401
+import logging
+import traceback
+
+from opentelemetry.instrumentation.django import DjangoInstrumentor
+from opentelemetry.trace import Span
+from django.core.exceptions import PermissionDenied  # pylint:disable=C0415
+
+from hypertrace.agent import constants
+from hypertrace.agent.filter.registry import Registry
+from hypertrace.agent.instrumentation import BaseInstrumentorWrapper
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=C0103
+
+
+class DjangoInstrumentationWrapper(BaseInstrumentorWrapper):
+    """wrapped class around django instrumentation"""
+    def instrument(self):
+        """configure django instrumentor w hooks"""
+        DjangoInstrumentor().instrument(request_hook=self.request_hook,
+                                        response_hook=self.response_hook)
+
+    def request_hook(self, span: Span, request):
+        """django request hook before request is processed by app"""
+        try:
+            body = request.body
+            span.update_name(f"{request.method} {span.name}")
+            self.generic_request_handler(request.headers, body, span)
+            full_url = request.build_absolute_uri()
```

Comment:
Does this trim query param? That is usually sensitive.

Reply:
No, just scheme + host + path